chopper/lib/src/http_logging_interceptor.dart

```diff
@@ -145,29 +145,33 @@ class HttpLoggingInterceptor
     }
 
     // Always start on a new line
-    chopperLogger.info('');
-    chopperLogger.info(
+    writeLine('');
+    writeLine(
       '<-- $reasonPhrase ${base.request?.method} ${base.request?.url.toString()}$bytes',
     );
 
     if (_logHeaders) {
-      base.headers.forEach((k, v) => chopperLogger.info('$k: $v'));
+      base.headers.forEach((k, v) => writeLine('$k: $v'));
 
       if (base.contentLength != null &&
           base.headers['content-length'] == null) {
-        chopperLogger.info('content-length: ${base.contentLength}');
+        writeLine('content-length: ${base.contentLength}');
       }
     }
 
     if (_logBody && bodyMessage.isNotEmpty) {
-      chopperLogger.info('');
-      chopperLogger.info(bodyMessage);
+      writeLine('');
+      writeLine(bodyMessage);
     }
 
     if (_logBody || _logHeaders) {
-      chopperLogger.info('<-- END HTTP');
+      writeLine('<-- END HTTP');
     }
 
     return response;
   }
+
+  void writeLine(String message) {
```

Comment:
Can you write this as an expression, i.e.
```dart
void writeLine(String message) => chopperLogger.info(message);
```

Reply:
Sure, done.